pkger/models.go

```diff
@@ -57,6 +57,10 @@ func (k Kind) OK() error {
 	return nil
 }
 
+func (k Kind) title() string {
```

Comment:
feels could be helpful if exported

Reply:
I think you're right. Although I think it could be premature to export right now since nothing is using it. I'll make this exported when this becomes useful outside the pkg here.